packages/api/src/external/commonwell-v2/document/cw-to-fhir.ts

```diff
@@ -71,12 +71,11 @@ export function cwToFHIR(
       system: doc.masterIdentifier?.system ?? undefined,
       value: doc.masterIdentifier?.value ?? docId,
     },
-    identifier:
-      doc.identifier?.map(id => ({
-        system: id.system ?? undefined,
-        value: id.value,
-        use: id.use ?? undefined,
-      })) ?? [],
+    identifier: (doc.identifier?.map(id => ({
+      system: id.system ?? undefined,
+      value: id.value,
+      use: id.use ?? undefined,
+    })) ?? []) as Identifier[],
```

Comment:
can we add the cast on the fallback value, instead of the whole mapping expression, please? That's safer